### DISCUSSION TITLE
azuracast installation on oracle Armv8

### DISCUSSION THREAD

**Discussion by surihost** [AzuraCast/AzuraCast#4241] [Q&A]
Hi,

Can we get the installation script to install azuracast on oracle cloud amrv8 instance with ubuntu 20.04 on it  ?

**Comment by surihost** [comment]
Hi,

Is it possible?

**Reply by Vaalyn** [reply]
We currently do not support installing AzuraCast via Docker on ARM systems.

**Reply by surihost** [reply]
OK,hope you can have an option soon to make it happen on arm system.

**Comment by surihost** [comment]


**Reply by Vaalyn** [reply]
You'll need to use the rolling-release version. The stable one is currently not working for Ansible since a dependency that was used in the stable version was using a dev branch that got merged by the upstream project and is thus not available anymore.

**Reply by surihost** [reply]
ok will try the rolling -release

**Reply by surihost** [reply]
Yes the installatio nwent succesfull now :)
But ssl is not activated during installation and dont see the command for that on post install page.Can you tell how to enable ssl?

**Reply by Vaalyn** [reply]
We do not include an automatic SSL for Ansible installations. You'll need to set this up yourself via for example Certbot.
